components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenuBuilder.kt

```diff
@@ -10,13 +10,14 @@ import android.content.Context
  * Helper class for building browser menus.
  *
  * @param items List of BrowserMenuItem objects to compose the menu from.
+ * @param extras Map of extra values that can be provided (e.g. 'customTab')
  */
 class BrowserMenuBuilder(
-    val items: List<BrowserMenuItem>
+    val items: List<BrowserMenuItem>,
+    val extras: Map<String, Any>? = null
```

Comment:
nit: If you turn this into `emptyMap()` then you do not need to deal with nullability.